features/manage_bookings.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Manage Bookings
+  In order to cancel a Booking
+  As a Manager
+  I want to manage and cancel booking
+
+ Scenario: A user can Cancel Booking by sending a DELETE request With Booking ID
+  Given I have booking with 
+    | start_time | 2021-09-29 10:00:00 |
+    | end_time   | 2021-09-29 10:30:00 |
+    | meeting_room_id | 1 |
+    | user_id | 1 |
```

Comment:
Arrange a table in the different manner - columns is the attributes and rows is the entries. It just gets easier to read and understand then this format.